tests/repl/imports.check

```diff
@@ -0,0 +1,24 @@
+scala> import scala.collection.mutable
+import scala.collection.mutable
+scala> val buf = mutable.ListBuffer[Int]()
+buf: scala.collection.mutable.ListBuffer[Int] = ListBuffer()
+scala> object o {
+     |   val xs = List(1, 2, 3)
+     | }
+defined module o
+scala> import o._
+import o._
+scala> buf += xs
+<console>:11: error: type mismatch:
+ found   : scala.collection.immutable.List[Int](o.xs)
+ required: String
+buf += xs
+       ^
+<console>:11: error: type mismatch:
```

Comment:
Not sure. I might depend on where the error is reported - same position in scalac, different ones in dotty. or some other artefact how += is compiled. But in any case `dotc` behaves like `doti`, so it's not a repl failure.